lib/lograge/log_subscribers/action_controller.rb

```diff
@@ -19,13 +19,34 @@ def unpermitted_parameters(event)
       private
 
       def initial_data(payload)
-        {
-          method: payload[:method],
-          path: extract_path(payload),
+        initial_data = {
           format: extract_format(payload),
-          controller: payload[:controller],
-          action: payload[:action]
+          http: {
```

Comment:
Should we include the `path` as well? If we make it a facet then we will be able to, for example, look at all requests to an endpoint across all environments.

Reply:
Path will automatically be extracted by Datadog from the URL